src/tools/tracejob.c

```diff
@@ -405,21 +404,20 @@ parse_log(FILE *fp, char *job, int ind)
 	strcpy(job_buf, job);
 
 	while (fgets(buf, buf_size, fp) != NULL) {
-		while(buf_size == strlen(buf) + incr){
+		while(buf_size == strlen(buf) + 1){
```

Comment:
One last little thing.  Could you put a space between the closing ) and the opening { here and on line 410?